src/server/filters.rs

```diff
@@ -203,6 +203,21 @@ async fn parse_toml<T: DeserializeOwned + Send>(buf: impl warp::Buf) -> Result<T
     toml::from_slice(&raw).map_err(|err| custom(BodyDeserializeError { cause: err.into() }))
 }
 
+pub fn json<T: DeserializeOwned + Send>() -> impl Filter<Extract = (T,), Error = Rejection> + Copy {
```

Comment:
There is a [built in](https://docs.rs/warp/0.3.1/warp/filters/body/fn.json.html) filter for parsing JSON. Any reason we need our own custom one?